src/python/pants/backend/python/subsystems/setup.py

```diff
@@ -200,6 +146,16 @@ class PythonSetup(Subsystem):
                 `python_test` / `python_tests`, and `pex_binary` which need to set a non-default\
                 resolve with the `resolve` field.
 
+            If a target can work with multiple resolves, you can either use the `parametrize`
+            mechanism or manually create a distinct target per resolve. See {doc_url("targets")}
+            about `parametrize`.
+
+            For example:
+
+                python_sources(
+                    resolve=parametrize("data-science", "web-app"),
+                )
```

Comment:
@thejcannon I could not find our discussion from last week about this. Do you remember if this is right? No need for back ticks?

Reply:
I never verified, but according to Markdown it's code because it's indented. The console will just dump whatever.
So depends on if we care about the console

Reply:
There are examples for how readme treats this from the docker backend (indented block is code).

> So depends on if we care about the console

What do you mean?
As you said, the console dumps whatever, so we don't have to "care about the console", just care about the experience the user has when reading this text on the console, which perhaps is what you meant? In which case I'm in favour of excluding the backticks to reduce the noise, on the console. (as the backticks does nothing for the markdown, when the block is also indented).

Reply:
https://github.com/pantsbuild/pants/blob/e8e22d590fe291c87418b293f3d89a9d6b6d8fa4/src/python/pants/backend/docker/subsystems/docker_options.py#L39-L53

https://www.pantsbuild.org/docs/reference-docker#section-registries

Reply:
@Eric-Arellano perhaps we want to record this doc nuance in the https://www.pantsbuild.org/docs/style-guide ? :)